code/datums/hud/human.dm

```diff
@@ -958,3 +958,10 @@
 			var/datum/hud/human/H = src.hud
 			H.update_status_effects()
 		return
+
+mob/proc/rest()
+	.= 0
```

Comment:
Would write as `. = 0` (i.e. with a space) if you want to do this way, but also `return 0` I think is slightly cleaner.